content/developer/cli.rst

```diff
@@ -292,9 +292,11 @@ Emails
 
 .. option:: --from-filter <address or domain>
 
-    Email address or domain that the mail server is configured for. If the 
-    email of the sender does not match this value, it might be encapsulated 
-    into ``mail.default.from``.
+    Define which email address the SMTP configuration will apply to. This field can be empty, a domain
```

Comment:
Awkward sentence `This field can...an entire email address.` , rearrange to this:

`The field can be a domain name or an entire email address, or it can remain empty.`